packages/api/src/providers/fitbit.ts

```diff
@@ -428,7 +422,8 @@ export class Fitbit extends Provider implements OAuth2 {
         extra: { context: "fitbit.revokeTokenFromExistingUsers", err, user: currentUser },
       });
       throw new Error(
-        `Failed to revoke Fitbit token and delete subscriptions for another user. User: ${currentUser}, Error: ${err}.`
+        `Failed to revoke Fitbit token and delete subscriptions for another user. User: ${currentUser}.`,
+        { cause: err }
```

Comment:
It's great that you're removing the error from the description, I'd suggest to remove the current user as well, so errors can be grouped together on Sentry - otherwise it will create an issue for each `currentUser`.

TBH, I think we can just rethrow the error, no need to create a new one and wrap it - we should be able to identify it came from this part of the code by looking at the stack trace.